composer.json

```
@@ -66,7 +66,22 @@
         "refresh": [
             "rm -rf composer.lock vendor html/core html/modules/contrib html/profiles/contrib html/themes/contrib",
             "@composer update -W --ansi"
-        ]
+        ],
+        "docker-up": [
+            "docker compose -f docker-compose.nginx.yml -p nginx up -d",
```

Lets allow this to fail in case the container is already running with that name.

```suggestion
            "docker compose -f docker-compose.nginx.yml -p nginx up -d || true",
```